package.json

```
@@ -23,24 +26,26 @@
 			"require": "./lib/index.js"
 		}
 	},
-	"main": "./lib/index.js",
+	"main": "lib/index.js",
 	"files": [
-		"lib"
+		"LICENSE.md",
+		"README.md",
+		"lib/",
+		"package.json"
 	],
 	"scripts": {
 		"build": "tsup",
-		"format": "prettier \"**/*\" --ignore-unknown",
+		"build:docs": "eslint-doc-generator",
```

Can we add the `pnpm build` back here?  Looks like we lost it from the prior version, as well as the `--rule-doc-title-format` option.

```suggestion
		"build:docs": "pnpm build && eslint-doc-generator --rule-doc-title-format",
```

---

Re `build`: I was actually happy to remove it. When I was tinkering with docs I found it inconvenient to have to wait for a full build -including type checking to create `.d.ts`  files` before seeing the updates.

Re `--rule-doc-title-format`: I moved it into `.eslint-doc-generatorrc.mjs`.

---

I specifically added it here, with reasoning for why it made sense at the time: https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/pull/955  

From the PR: 
>  There's never really a case where you'd not want to have a build happen when running the doc update command, and when running locally, it's not super obvious that it's necessary to run a build first. Baking it into the update script removes that ambiguity.

---

`eslint-doc-generator` only works off of what's in the dist folder.  So if you don't run `build` then you're either not working off of the latest, or you've already built it manually at some point.

---

Totally cool with moving the doc title config to the rc file.  I didn't notice that. 👍

---

Ah right, thanks. I'm thinking we can get around the type generation slowness by just having it run `--no-dts`. So:

```json
"build:docs": "pnpm build --no-dts; eslint-doc-generator",
```

How does that sound to you?

---

Yeah, that's a nice in between 👍